console/src/app/pages/events/events.component.html

```diff
@@ -0,0 +1,121 @@
+<div class="max-width-container">
+  <h1 class="events-title">{{ 'IAM.EVENTS.TITLE' | translate }}</h1>
+  <p class="events-desc cnsl-secondary-text">{{ 'IAM.EVENTS.DESCRIPTION' | translate }}</p>
+
+  <cnsl-refresh-table
+    [hideRefresh]="true"
+    (refreshed)="refresh()"
+    [dataSize]="dataSource.data.length"
+    [loading]="_loading | async"
+  >
+    <div actions>
+      <cnsl-filter-events (requestChanged)="filterChanged($event)"></cnsl-filter-events>
+    </div>
+
+    <table
+      [dataSource]="dataSource"
+      mat-table
+      class="table views-table"
+      aria-label="Views"
+      matSort
+      (matSortChange)="sortChange($event)"
+    >
+      <ng-container matColumnDef="editor">
+        <th mat-header-cell *matHeaderCellDef>{{ 'IAM.EVENTS.EDITOR' | translate }}</th>
+        <td mat-cell *matCellDef="let event">
+          <ng-container *ngIf="event | toobject as event">
+            <div class="editor-row" *ngIf="event.editor as editor">
+              <!-- <cnsl-avatar
+              *ngIf="editor && editor.displayName; else cog"
+              class="avatar"
+              [name]="editor.displayName"
+              [avatarUrl]="editor.avatarUrl || ''"
+              [forColor]="editor.preferredLoginName ?? editor.displayName"
+              [size]="32"
+            >
+            </cnsl-avatar>
+            <ng-template #cog>
+              <cnsl-avatar [forColor]="editor?.preferredLoginName ?? 'franz'" [isMachine]="true">
+                <i class="las la-robot"></i>
+              </cnsl-avatar> </ng-template
+            > -->
+
+              <span class="name" *ngIf="editor.displayName">{{ editor.displayName }}</span>
+              <span class="state" *ngIf="editor.service">{{ editor.service }}</span>
+            </div>
+          </ng-container>
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="aggregate">
+        <th mat-header-cell *matHeaderCellDef>{{ 'IAM.EVENTS.AGGREGATE' | translate }}</th>
+        <td mat-cell *matCellDef="let event">
+          <ng-container *ngIf="event | toobject as event">
+            <div class="aggregate-row">
+              <span class="id">{{ event.aggregate.resourceOwner }}</span
```

Comment:
shouldn't this be the aggregate id?

Reply:
i guess we should display the resourceOwner in the dialog though

Reply:
yes.. fixed 👍